nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-controller-service.js

```diff
@@ -402,6 +404,36 @@
         });
     };
 
+    var updateValidationErrors = function (validationErrorIcon, referencingComponent) {
```

Comment:
This function appears to be a duplicate. Likely another mixed up with the rebase. Also, there's another condition that may have been missed. Search for 'ParameterProvider' to find other instances.